jflex/src/main/java/jflex/Main.java

```diff
@@ -10,21 +10,22 @@
 package jflex;
 
 import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
 import java.io.IOException;
 import java.lang.reflect.Field;
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Map;
 import java.util.SortedMap;
 import java.util.SortedSet;
 import java.util.TreeMap;
 import java.util.TreeSet;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import jflex.gui.MainFrame;
 import jflex.unicode.UnicodeProperties;
+import org.apache.commons.cli.CommandLine;
```

Comment:
This seems mixed with the other PR on changing the CLI.

Reply:
Yes, I started here when I refactored the (jflex) Options, and then realized that the commons-cli change was unrelated and could be moved in a different channge (now PR #259)

Reply:
Got it.